src/app/app.component.spec.ts

```diff
@@ -155,4 +158,68 @@ describe('AppComponent', () => {
        flush();
   }));
 
+  // spinner is shown when user is authorized
+
+  it('should hide the spinner, after workspace is (intially) loaded', fakeAsync(() => {
+    resetCalls(mockNg4SpinnerService);
```

Comment:
is this necessary? The test bed setup seems to create a fresh instance `beforeEach` test.